ios/RNSpokestack.m

```diff
@@ -195,78 +178,69 @@ - (void)failureWithNluError:(NSError * _Nonnull)nluError {
+    self.speechConfig.wakeActiveMax = ([config valueForKeyPath:@"pipeline.wake-active-max"]) ? [RCTConvert NSInteger:[config valueForKeyPath:@"pipeline.wake-active-max"]] : self.speechConfig.wakeActiveMax;
+    self.speechConfig.frameWidth = ([config valueForKeyPath:@"pipeline.frame-width"]) ? [RCTConvert NSInteger:[config valueForKeyPath:@"pipeline.frame-width"]] : self.speechConfig.frameWidth;
+    self.speechConfig.preEmphasis = ([config valueForKeyPath:@"pipeline.pre-emphasis"]) ? [[RCTConvert NSNumber:[config valueForKeyPath:@"pipeline.pre-emphasis"]] floatValue] : self.speechConfig.preEmphasis;
+    self.speechConfig.filterModelPath = ([config valueForKeyPath:@"pipeline.wake-filter-path"]) ? [RCTConvert NSString:[config valueForKeyPath:@"pipeline.wake-filter-path"]] : self.speechConfig.filterModelPath;
+    self.speechConfig.encodeModelPath = ([config valueForKeyPath:@"pipeline.wake-encode-path"]) ? [RCTConvert NSString:[config valueForKeyPath:@"pipeline.wake-encode-path"]] : self.speechConfig.encodeModelPath;
+    self.speechConfig.detectModelPath = ([config valueForKeyPath:@"pipeline.wake-detect-path"]) ? [RCTConvert NSString:[config valueForKeyPath:@"pipeline.wake-detect-path"]] : self.speechConfig.detectModelPath;
+    self.speechConfig.wakeThreshold = ([config valueForKeyPath:@"pipeline.wake-threshold"]) ? [[RCTConvert NSNumber:[config valueForKeyPath:@"pipeline.wake-threshold"]] floatValue] : self.speechConfig.wakeThreshold;
+    self.speechConfig.wakewords = ([config valueForKeyPath:@"properties.wakewords"]) ? [RCTConvert NSString:[config valueForKeyPath:@"properties.wakewords"]] : self.speechConfig.wakewords;
+    self.speechConfig.wakewordRequestTimeout = ([config valueForKeyPath:@"properties.wake-request-timeout"]) ? [RCTConvert NSInteger:[config valueForKeyPath:@"properties.wake-request-timeout"]] : self.speechConfig.wakewordRequestTimeout;
+
+    /// MARK: Spokestack configuration
 
-    self.speechConfig.apiId = ([config valueForKeyPath:@"tts.api-id"]) ? [RCTConvert NSString:[config valueForKeyPath:@"properties.api-id"]] : self.speechConfig.apiId;
-    self.speechConfig.apiSecret = ([config valueForKeyPath:@"tts.api-id"]) ? [RCTConvert NSString:[config valueForKeyPath:@"tts.api-secret"]] : self.speechConfig.apiSecret;
+    self.speechConfig.tracing = ([config valueForKeyPath:@"properties.trace-level"]) ? [RCTConvert NSInteger:[config valueForKeyPath:@"properties.trace-level"]] : self.speechConfig.tracing;
+    self.speechConfig.apiId = ([config valueForKeyPath:@"properties.api-id"]) ? [RCTConvert NSString:[config valueForKeyPath:@"properties.api-id"]] : self.speechConfig.apiId;
+    self.speechConfig.apiSecret = ([config valueForKeyPath:@"properties.api-id"]) ? [RCTConvert NSString:[config valueForKeyPath:@"properties.api-secret"]] : self.speechConfig.apiSecret;
```

Comment:
There's a mismatch here: checking for `api-id` and using `api-secret`. Luckily, it's the mostly benign kind of mismatch.

Reply:
nice catch, copypasta strikes again